domains/transfer/Transfer.API/API.IntegrationTests/Factories/TransferAgreementsApiWebApplicationFactory.cs

```diff
@@ -159,20 +154,20 @@ public async Task SeedTransferAgreementProposals(IEnumerable<TransferAgreementPr
     public HttpClient CreateUnauthenticatedClient()
     {
         var client = CreateClient();
-        client.DefaultRequestHeaders.Add("EO_API_VERSION", "20231123");
+        client.DefaultRequestHeaders.Add("EO_API_VERSION", "20240103");
```

Comment:
Would be nice with this static class with known versions, instead of magic strings all over the codebase.

Reply:
I was thinking about doing that in a separate commit. There is already alot happening in this commit.

Reply:
Done:

https://github.com/Energinet-DataHub/energy-origin/pull/684/commits/761c9e64fb855ccc8e28e847b5c2986e0c35c998